report_generator/README.md

```diff
@@ -2,10 +2,14 @@
 
 Report generator produces Excel table file from json benchmark log files.
 
-Run `python report_generator.py --result-files bench_log_1.json,bench_log_2.json [--report-file new_report.xlsx --generation-config gen_config.json --merging none]` to launch report generation.
+Run `python report_generator.py --result-files bench_log_1.json,bench_log_2.json [--report-file new_report.xlsx --generation-config default_report_gen_config.json]` to launch report generation.
 
 runner options:
 * ``result-files`` : comma-separated benchmark json result file paths
 * ``report-file`` : report file path
 * ``generation-config`` : generation configuration file path
-* ``merging``: *full*, *none*, *sw_only*, *hw_only*. How to merge same cases in benchmark logs
+
+config parameters:
+* ``header`` : Column names in the table header. These parameters are also used to compare reports. If the name is compound, then it should be separated by the '':'' symbol.
+* ``comparison_method`` : Comparison of two results. The options are: ``1 operation 2`` or ``2 operation 1``. Default is ``2 / 1``.
+* ``aggregation_metrics`` : Metric applied to columns with comparisons of two reports. For each of these metrics, a separate sheet with a summary is compiled. It is important that the function is in Excel. For example: ``geomean`` or ``average``.

```

Comment:
```suggestion
* ``aggregation_metrics``: The metrics applied to the columns with the comparisons of two reports. For each of these metrics, a separate sheet with a summary is compiled. Use Excel function names to provide metrics, for example: ``geomean`` or ``average``.
```

If users want to provide multiple metrics, they should provide them as a list, I assume? I think you'd benefit from having examples of how the config file could look like.

Reply:
```
"aggregation_metrics": [
        "geomean",
        "average"
    ]
```